### DISCUSSION TITLE
How can I offer downloads on the blog?

### DISCUSSION THREAD

**Discussion by ghost** [datenstrom/community#282] [Ask a question] [answered]
Hello Developers,

I would like to offer in my Yellow-Blog on a page downloads. How can I do this and which directory in the FTP account are uploaded the ZIP files?

Thanks in advance.

**Comment by schulle4u** [comment, marked as answer]
Usually all non-yellow related files are stored in the `/media` directory and it's subfolders. Do you want a automatically generated list of media files to be displayed on your site? The examples in the "Yellow Files" section of the [API documentation](https://developers.datenstrom.se/help/api) might give you a starting point. 

If you want some descriptive text for each of your downloads actually you can simply drop your files somewhere in the `/media` folder and create a link to it: `[Download example.zip](/media/downloads/example.zip)`

Steffen

**Comment by ghost** [comment]
Thank you, it worked just fine and so I offer my first download over a yellow page 👍
